Add Chinese, German and Spanish translations

huey735 opened the pull request:
I translated the images in the DAO page and renamed the image files accordingly.

huey735 committed:
Fix typos

Fix typos and remove unnecessary strings from
the translation files

huey735 committed:
Add Chinese, German and Spanish translations

m52go commented on _layouts/default.html:
```diff
@@ -733,7 +733,7 @@
 
 <!--Select includes/main_nav according to page lang or absence of lang-->
-          {% if page.lang == "pt-PT"  %}
+          {% if page.lang == "de" or page.lang == "es" or page.lang == "pt-PT" or page.lang == "zh-CN" %}
```
To avoid adding languages to the condition every time...with lots of languages, this will quickly get unwieldy.

How about this?

```
{% if page.lang == "en" %}
    {% include main_nav.html %}
{% else %}
    {% include main_nav_tr.html %}
{% endif %}
```

huey735 replied:
I reworked it to look like this:
```
          {% if page.lang == "en" or page.ref == "blog" %}
            {% include main_nav.html %}
          {% else %}
            {% include main_nav_tr.html %}
          {% endif %}
```
Otherwise the Blog page doesn't render a nav bar.

m52go commented on _data/es.yml:
```diff
@@ -0,0 +1,344 @@
+  pJoin: >
+    <p>Bisq siempre está buscando desarrolladores altamente cualificados, diseñadores y expertos en seguridad para unirse a nuestra <a href="localhost:4001/community/">comunidad</a>. Si está interesado en ayudar a mejorar Bisq, hay muchas maneras de<a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>.
+     Nos encantaría<a href="localhost:4001/community/">saber de ti</a>!</p>
+    <p>Nuestra <a href="localhost:4001/roadmap/">hoja de ruta</a> proporciona una vista general de las características y hitos hasta llegar a la v.1.0. de Bisq</p>
```
Looks like the period at the end of the sentence is misplaced:

`hasta llegar a la v.1.0 de Bisq.`

m52go commented on _data/es.yml:
```diff
@@ -0,0 +1,344 @@
+    <li class="mt-0 mb-4 pb-4 border-bottom step step-text-2" data-image="step-image-2">Añada sus cuentas de moneda nacional o cripto</li> <li class="mt-0 mb-4 pb-4 border-bottom step step-text-3" data-image="step-image-3">Cree una nueva oferta o tome una existente.</li> <li class="mt-0 step step-text-4" data-image="step-image-4">Siga la interfaz de usuario para finalizar el intercambio.</li>
+  hJoin: Unirse a la comunidad
+  pJoin: >
+    <p>Bisq siempre está buscando desarrolladores altamente cualificados, diseñadores y expertos en seguridad para unirse a nuestra <a href="localhost:4001/community/">comunidad</a>. Si está interesado en ayudar a mejorar Bisq, hay muchas maneras de<a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>.
```
I don't know the language but it seems like there are missing spaces around the links here.

Notice "maneras **decontribuir**" and "Nos **encantaríasaber** de ti".

![Screenshot from 2019-08-08 00-28-38](https://user-images.githubusercontent.com/735155/62675205-fc7def00-b973-11e9-81e5-c6d986fafb7d.png)

m52go commented on _data/de.yml:
```diff
@@ -0,0 +1,316 @@
+  seeMore: >
+    <p>Weitere Ressourcen finden Sie in unserem <a href='https://docs.bisq.network/dao.html'>Alles über die Bisq DAO</a> Dokument.</p> <p>Bisherige Mitwirkende finden in <a href='https://docs.bisq.network/compensation.html#get-added-to-genesis'>diesem Link</a> Anweisungen, wie sie der Genesis BSQ-Distribution hinzugefügt werden können.</p>
+downloads:
+  hBisqDownloads: Bisq Downloads
```
runningWindows
runningMac
runnigDebian
runningRedHat

Need to remove the periods here, otherwise, they result in a double period. Also check pt-PT.yml. I think the others are fine.

The runningOS keys in the `homepage_content` section can have periods, but the runningOS keys in the `downloads` section shouldn't. 

If you want to keep it simple you could just remove periods from all runningOS keys...the `homepage_content` doesn't really need periods anyway.

huey735 replied:
Can you show me where the strings for the "running" keys appear on the website? I guess it isn't detecting on my computer.

m52go replied:
Yeah it's on the Downloads page at the end of the sentence where it says "You appear to be using..."

This is ok, no double period:

![Screenshot from 2019-08-08 04-21-24](https://user-images.githubusercontent.com/735155/62687115-13ccd480-b994-11e9-8dab-9f160dc0479d.png)

This has a double period:

![Screenshot from 2019-08-08 04-20-05](https://user-images.githubusercontent.com/735155/62686987-d49e8380-b993-11e9-85ee-92893ac5cec6.png)

Only a problem for de and pt-pt though...you won't see it for any other language.

huey735 replied:
Oh, thanks. Chrome doesn't display it for some reason here, but it does show up on Firefox.

m52go reviewed (commented):
Looks good to me...couple of minor things and it's ready to go.

huey735 committed:
Fix punctuation, empty spaces and typos

huey735 committed:
Rework control flow to include nav bar

I reworked the Liquid control flow tags to include the nav bar.
Now, pages where "lang: en" or "ref: blog" will include
the English nav bar while everywhere else will include a nav
bar according to its language.

huey735 committed:
Rework control flow to include footer

m52go commented on _data/es.yml:
```diff
@@ -0,0 +1,344 @@
+  pJoin: >
+    <p>Bisq siempre está buscando desarrolladores altamente cualificados, diseñadores y expertos en seguridad para unirse a nuestra <a href="localhost:4001/community/">comunidad</a>. Si está interesado en ayudar a mejorar Bisq, hay muchas maneras de <a href="https://docs.bisq.network/contributor-checklist.html" target="_blank" rel="noopener">contribuir</a>.
+     Nos encantaría <a href="localhost:4001/community/">saber de ti</a>!</p>
+    <p>Nuestra <a href="localhost:4001/roadmap/">hoja de ruta</a> proporciona una vista general de las características y hitos hasta llegar a la v.1.0. de Bisq.</p>
```
```suggestion
    <p>Nuestra <a href="localhost:4001/roadmap/">hoja de ruta</a> proporciona una vista general de las características y hitos hasta llegar a la v.1.0 de Bisq.</p>
```

huey735 committed:
Correct punctuation

Co-Authored-By: Steve Jain <mfiver@gmail.com>
